README.md

```diff
@@ -5,18 +5,30 @@
 # OpenShift Ansible
 
 This repository contains [Ansible](https://www.ansible.com/) roles and
-playbooks to install, upgrade, and manage
-[OpenShift](https://www.openshift.com/) clusters.
-
-**Note**: the Ansible playbooks in this repository require an RPM
-package that provides `docker`. Currently, the RPMs from
-[dockerproject.org](https://dockerproject.org/) do not provide this
-requirement, though they may in the future. This limitation is being
-tracked by
-[#2720](https://github.com/openshift/openshift-ansible/issues/2720).
-
-## Getting the correct version
-When choosing an openshift release, ensure that the necessary origin packages
+playbooks to install, upgrade, and manage the Origin Kubernetes Distribution in a fully managed container orchestration platform, on an existing computing infrastructure.  For an introduction to openshift [OpenShift](https://www.openshift.com/).
```

Comment:
`Origin Kubernetes Distribution` is not a legally correct name, please use Openshift instead (as these playbooks could be used for OCP and OKD)